misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenConstants.java

```diff
@@ -43,12 +43,13 @@ private BindgenConstants() {
     public static final String CONSTANTS_FILE_NAME = "Constants.bal";
     public static final String ARRAY_BRACKETS = "[]";
     public static final String MODULES_DIR = "modules";
+    public static final String BALLERINA_TOML = "Ballerina.toml";
```

Comment:
Cant we use `ProjectConstants.BALLERINA_TOML` without redefining a new cont?

Reply:
Thanks, fixed it.